### PR TITLE
NETOBSERV-273 Reducing empty array with no initial value

### PR DESCRIPTION
**Pull request by jpinsonneau**
Fix summary reduce error when panel is kept open during refresh

https://issues.redhat.com/browse/NETOBSERV-273

### CODE REVIEW

File: web/src/components/query-summary/query-summary.tsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const QuerySummaryContent: React.FC<{
         <Tooltip content={<Text component={TextVariants.p}>{t('Filtered sum of packets')}</Text>}>
           <Text id="packetsCount" component={TextVariants.p}>
             {t('{{count}} packets', {
-              count: flows.map(f => f.fields.Packets).reduce((a, b) => a + b)
+              count: _.isEmpty(flows) ? 0 : flows.map(f => f.fields.Packets).reduce((a, b) => a + b)
```
Comment (jotak) [inline web/src/components/query-summary/query-summary.tsx:60]: that's fine, however writing `reduce((a, b) => a + b, 0)` should do the same and is much more idiomatic javascript. But lgtm

**Comment by jotak** [comment]
/lgtm
(not tested)

File: web/src/components/query-summary/query-summary.tsx (hunk 1 above)
Comment (jpinsonneau) [reply web/src/components/query-summary/query-summary.tsx:60]: I'm not used to `reduce` function. Let's move to this then. 
Thanks @jotak

**Comment by jpinsonneau** [comment]
Do you want to take the time to test it before merge @jotak ?

**Comment by jotak** [comment]
I won't have the time today .. I guess @memodi or @brahaney can pick it after merge (it's a very small change)

**Pull request merged**